core/src/main/java/org/alfresco/util/TempFileProvider.java

```diff
@@ -485,26 +481,19 @@ private static int removeFiles(File directory, long removeBefore, long longLifeB
             int count = 0;
             for (File file : filesToIterate)
             {
+                if (shouldTheDeletionStop())
+                {
+                    break;
+                }
                 if (file.isDirectory())
                 {
-                    if(isLongLifeTempDir(file))
-                    {
-                        // long life for this folder and its children
-                        int countRemoved = removeFiles(file, longLifeBefore, longLifeBefore, true);  
-                        if (logger.isDebugEnabled())
-                        {
-                            logger.debug("Removed " + countRemoved + " files from temp directory: " + file);
-                        }
-                    }
-                    else
-                    {
-                        // enter subdirectory and clean it out and remove itsynetics
-                        int countRemoved = removeFiles(file, removeBefore, longLifeBefore, true);
-                        if (logger.isDebugEnabled())
-                        {
-                            logger.debug("Removed " + countRemoved + " files from directory: " + file);
-                        }
-                    }
+                    // long life for this folder and its children
+                    // OR
+                    // enter subdirectory and clean it out and remove itsynetics
+                    int countRemoved = removeFiles(file,
+                        isLongLifeTempDir(file) ? longLifeBefore : removeBefore, longLifeBefore,
+                        true);
+                    logger.debug("Removed " + countRemoved + " files from temp directory: " + file);
```

Comment:
this changes removes the ability to know in the logs if we have along lived file or not...

Reply:
Addressed.